proposals/lip-0052.md

```diff
@@ -1828,6 +1831,9 @@ def transferCrossChain(
     includeAttributes: bool
 ) -> None:
 
+    if receivingChainID == OWN_CHAIN_ID:
```

Comment:
I just realized that in this function in all other checks we emit failure events, so perhaps we should do this here as well for consistency (same for the token module..). Not sure how important it is though, espcecially for the token module..

Reply:
I agree with the suggested changes and have pushed the related commit. I will also expand the Description and Proposed solution accordingly.